Assets/Scripts/Augment/GunBody.cs

```diff
@@ -12,4 +12,33 @@ public class GunBody : MonoBehaviour
 
     // Where to attach barrel
     public Transform attachmentSite;
+
+    [SerializeField, Range(0,1)]
+    protected float reloadEfficiencyPercentage = 1f;
+
+    //TODO: Modifier refactor
+    protected GunController gunController;
```

Comment:
Elaborate? I assume you mean that we should give access to the gun controller in some other way?

Reply:
There is bad coupling between gunController and augments. I'll refactor that in another PR